packages/angular_devkit/schematics/tasks/package-manager/install-task.ts

```diff
@@ -45,6 +47,9 @@ export class NodePackageInstallTask implements TaskConfigurationGenerator<NodePa
       if (options.packageName != undefined) {
         this.packageName = options.packageName;
       }
+      if (options.allowScripts != undefined) {
```

Comment:
Nit: Should these be using `!==`? Probably not an issue when comparing with `undefined`, but good to use it consistently.